src/LightSpeed/spawnTopic.js

```diff
@@ -0,0 +1,34 @@
+/* exported LSSpawnTopic */
+
+const {GObject, Gtk} = imports.gi;
+
+const {LSUserFunction} = imports.LightSpeed.userFunction;
+
+var LSSpawnTopic = GObject.registerClass({
+    Properties: {
```

Comment:
@ptomato Adding this here doesn't work because then you bind the signal on the TopicButton to it instead of to the property in the UserFunction that emits the 'needs-attention' signal. However, I don't know what the fix is here, since it seems like doing multiple bind_properties to one TopicButton doesn't work because the last call to bind_property takes precedence. I suspect you can figure out how to do it, though.